examples/docker-compose.yml

```diff
@@ -1,16 +1,14 @@
 version: '2'
 services:
   conjur:
-    image: registry.tld/conjur-appliance-cuke-master:4.9-stable
+    image: registry2.itci.conjur.net/conjur-appliance-cuke-master:4.9-stable
```

Comment:
This should stay `registry.tld`, let's not hardcode in our internal endpoints. Jenkins is already set up that way, and it's just a line in `/etc/hosts` for us. 

https://github.com/conjurinc/docker-registry-ansible-aws#usage